test/quantization/fx/test_model_report_fx.py

```diff
@@ -1033,7 +1034,7 @@ def test_generate_report(self):
 
             # prepare and callibrate two different instances of same model
             # prepare the model
-            example_input = torch.randn(1, 3, 3, 3)
+            example_input = model_full.get_example_inputs()[0]
```

Comment:
this should be a tuple of inputs (e.g. example_inputs = (torch.randn(1, 3, 3, 3),))

Reply:
is a tuple of inputs where we get it from the actual class